Add 'Replace File' button to item details drawer for files (#19731)
directus/directus: .changeset/empty-berries-fly.md

---
"@directus/app": patch
---

> Added ability to replace files in the item details drawer

Reviewer: ```suggestion
Added ability to replace files from within item details drawer
```
